lib/endpoints/accesskeys.js

```diff
@@ -21,20 +22,47 @@ var resources = require('../resources');
 var sprintf = util.format;
 
 var InvalidArgumentError = restify.InvalidArgumentError;
+var ForbiddenError = restify.ForbiddenError;
+var InternalError = restify.InternalError;
+
+var MAX_KEYS = 100;
```

Comment:
I was gonna ask if this value (100) was for a specific reason; saw its use below as a max-keys-for-a-user value. A comment here either citing a standard or language like "seems like a good figure to us" would help a new-to-this-code reader.

Reply:
My thought was that 100 should be more than enough for most use-cases while also preventing a for loop from flooding UFDS. It's worth noting that AWS only permits **3** of similar type keys (but that does feel unnecessarily restrictive.)

I'll add that info as comment here but I'm also open to adjusting the value if we think its necessary.

Reply:
Resolving.